Makefile

```diff
@@ -40,18 +44,22 @@ deps:
 	go get -u github.com/goadesign/gorma
 
 generate: $(DESIGNS)
-	goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
-	goagen js -d ${PACKAGE_NAME}/${DESIGNDIR} -o assets/ --noexample
-	goagen gen -d ${PACKAGE_NAME}/${DESIGNDIR} --pkg-path=github.com/goadesign/gorma
-	go-bindata-assetfs -debug assets/...
-	godep get
+	$(GOPATH)/bin/goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
```

Comment:
Don't care. The point is, GOPATH/Workspaces/Versions etc should be controlled by the dev outside of our script. Rely on standard GO type env variables. Don't build a one of go setup into our scripts.